uploadNeuroDB/imaging_upload_file.pl

```diff
@@ -337,6 +359,59 @@ =head2 Methods
             . "inserted into the database \n";
 spool($message,'N', $notify_notsummary);
 
+################################################################
+############### Email last completion message ##################
+################################################################
+my @inserted_files = getInsertedFileNamesUsingUploadID($upload_id);
+
+$message = "\n$minc_created minc file(s) created, "
+            . "and $minc_inserted minc file(s) "
+            . "inserted into the database: \n";
+open MAIL, "| mail $mail_user";
+print MAIL "Subject: IMAGING_UPLOAD_FILE: ".$uploaded_file." insertion completed.\n\n";
+print MAIL "Progress: Success\n";
+print MAIL $message."\n";
+print MAIL join("\n", @inserted_files)."\n";
+close MAIL;
+
+################################################################
+############### getInsertedFileNamesUsingUploadID###############
+################################################################
+=pod
+
+=head3 getInsertedFileNamesUsingUploadID($upload_id)
+
+Function that gets names of inserted files using the upload ID
+
+INPUT: The upload ID
+
+RETURNS: The array of inserted file names
+
+=cut
+
+
+sub getInsertedFileNamesUsingUploadID {
+
+    my $upload_id = shift;
+    my ( $file_names, $query ) = '';
```

Comment:
```suggestion
    my (@file_names, $query );
```
No need to initialize it to empty string here